[Blog] Developing a meeting culture that your employees will love (#71)
rathesDot/rathes.me: src/content/articles/en/developing-meeting-culture-employees-love.mdx

description: Do you like meetings? Do your employees? Creating a meeting culture that is efficient and that everybody loves isn't that difficult if you follow these ideas.
categories:
  - Business
tags:
  - Meetings
  - Company Culture
locale: en_US
---

import Note from "../../../components/Note"
import Calculation from "../../../components/Calculation"

# Developing a meeting culture that your employees will love

Be honest to yourself. Do you like meetings in your business? Most of the people I talk to really hate meetings. They are unproductive, they are time-consuming, they are tiring. But if you dig deeper, the contents of those meetings are unavoidable. It is the meetings that lead to frustrations.
Let me provide you a way how you can improve your meetings culture so that your employees, and colleagues won’t hate them anymore.

<Note>
  Note: The following ideas come from experiences working in and with
  distributed teams but you can apply all of them to co-located teams as well.
</Note>

## 1. Do you need a meeting for this?

Nobody likes meetings, but some meetings are crucial to success. It is just that not everything needs to be discussed in a meeting.

> Do you need to inform someone? Or multiple people? Why not send them are e-mail? Or a message in Slack? That is way faster and since it is async it will not interrupt the people you are informing.

Author: ```suggestion
Do you need to inform someone? Or multiple people? Why not send them an e-mail? Or a message in Slack? That is way faster and since it is asynchronous it will not interrupt the people you are informing.
```

> Make sure that your company has a platform for urgent communication: And make sure everybody in that company is aware of it.

Author: ```suggestion
Make sure that your company has a platform for urgent communication. And make sure everybody in that company is aware of it.
```

> For example, the general channel could be a good place. Ensure that is only used for announcements that are important (and define what important is) and also make sure that everybody checks the channel on a regular defined basis (like once or twice a day).

Author: ```suggestion
For example, the #general in Slack channel could be a good place. Ensure that it is only used for announcements that are important (and define what important is) and also make sure that everybody checks the channel on a defined regular basis (like once or twice a day).
```

> Another example of meetings that can be avoided are presentations: Your weekly results presented to the entire staff taking two hours, are they really necessary? Why not just write a note regularly and publish them?

Author: ```suggestion
Another example of meetings that can be avoided are presentations: Your weekly results presented to the entire staff taking two hours, are they really necessary? Why not write a note regularly and publish them?
```

A company-wide internal blog would be a perfect place for that. People can read them when they are interested and they have time (make sure they have) and also provide feedback.
What about meetings to discuss things? While urgent and important discussion like creating an offer to your potential next CMO is worth a meeting it doesn’t make sense to get 48 people into a meeting room to discuss possible destinations for your upcoming team trip.

> Offload such a discussion to an asynchronous communication channel. There you can let the discussion grow. Once in a while, you go ahead and move the discussion in the right direction and you eventually get what you want. Or maybe not, but then you didn’t waste the time of 48 people.

Author: ```suggestion
Offload such a discussion to an asynchronous communication channel. There, you can let the discussion grow. Once in a while, you go ahead and move the discussion in the right direction and you eventually get what you want. Or maybe not, but then you didn’t waste the time of 48 people.
```

There are a lot of other things that can be done asynchronously like stands-ups or grooming in Scrum. In teams that work together across multiple time zones, there is even no other way. If things are prepared properly by everyone, it works perfectly fine.

## Meetings you can not avoid

While I’m confident enough that you can handle every topic asynchronously, there might be one or the other situation that requires people sitting together.

> It few ideas will make sure that those meetings are as pleasant as possible

Author: ```suggestion
The following few ideas will make sure that those meetings are as pleasant as possible
```

### 2. Plan meetings early enough

You won’t have that many meetings anymore but make sure that the few you do have are planned as early as possible.
Never plan a meeting 30 minutes before it actually takes place. Nobody, including yourself, will have enough time to prepare for it.

> Decide upfront with your staff on how early a meeting needs to be planned upfront. In some companies 24 hours one enough. Some others say 2 days or even more: Try out and what fits your company best.

Author: ```suggestion
Decide upfront with your staff on how early a meeting needs to be planned upfront. In some companies, 24 hours are enough. Some others say 2 days or even more: Try out and what fits your company best.
```

#### Never touch other people's agendas

> Important to note is that you never place a meeting on someone else's calendar. Instead your invite them to a meeting and give them the opportunity to check for themselves if they can actually make it.

Author: ```suggestion
Important to note is that you never place a meeting on someone else's calendar. Instead, you invite them to a meeting and give them the opportunity to check for themselves if they can actually make it.
```

While there might be an empty spot in two days at 9 am on your colleague's agenda, you don’t know if they actually will find some time to prepare for it before.

### 3. Minimize the number of attendees

I have never seen a meeting with 48 people in a room that was productive and had the expected outcome.
Ask yourself who the bare minimum of people are to get that job done and only invite them. The fewer people attend a meeting the less time you spend on it.
A 1-hour meeting with 48 people is 48 hours of time not only one hour.

#### Nobody has to attend the meeting

Even with the minimum amount of people attending, make sure that those who couldn’t make it do not block you from having this meeting.
That also means that you have the meetings only to discuss things, not to decide. The decision will be based on the outcomes of that meeting but will be made after the meeting. That will ensure that the people who missed out have a chance to get up-to-date and be part of the decision.

### 4. No meeting without a desired outcome and agenda

> When you plan a meeting, ask yourself “Why do I need this meeting What do I want to achieve with it?”

Author: ```suggestion
When you plan a meeting, ask yourself “Why do I need this meeting? What do I want to achieve with it?”
```

> That desired outcome will define the topic of your meeting and based on that you can create an agenda for the meeting things that need to be discussed.

Author: ```suggestion
That desired outcome will define the topic of your meeting and based on that you can create an agenda for the meeting, things that need to be discussed.
```

You should have these two prepared before you send out invites because the outcome and agenda will be the information upon which the attendees can decide if they need to be in the meeting.
The agenda will also be the base for everybody attending to prepare the meeting.

### 5. Every meeting has a fixed duration

Did you ever plan a meeting and wondered how long that meeting should take place? Well, now you won’t anymore.
The length of each meeting can be calculated by the following formula:

<Calculation># attendees x 10 minutes + 15 minutes</Calculation>

The 10 minutes and 15 minutes can be adapted to your company but I found that those numbers are pretty much suitable for most of the companies.

#### Each attendee has a speaking slot

The first part of the formula foresees a speaking slot of 10 minutes for everyone attending. During that time nobody else speaks but the one in turn.

#### Q + A

> The last 15 minutes a reserve for a Q + A with "A" being "answers" and "actions points"

Author: ```suggestion
The last 15 minutes are reserved for a Q + A with "A" being "answers" and "actions points"
```

During that time the initiator of the meeting takes the role of the moderator.
If you need longer than this time for your meeting that means your agenda is too big or people didn’t prepare property (see following section).

### 6. Everybody prepares for and reviews the meeting

Once every attendee has accepted their invite to the meeting they are required to prepare for the meeting. Do the research, note down your ideas or questions.
If there is something that you want others to check upfront, make sure that you communicate this early enough. You won’t have time to show a 20-minute video during the meeting.
After the meeting has taken place, the initiator collects everything that was discussed and provides the meeting notes to the knowledge base.

> This can be made easier if you record meetings. but videos should never replace written notes because you need more time to consume videos than notes.

Author: ```suggestion
This can be made easier if you record meetings, but videos should never replace written notes because you need more time to consume videos than notes.
```
